tests/suites/host_test.function

```diff
@@ -447,6 +451,8 @@ static void write_outcome_result( FILE *outcome_file,
                                      i == 0 ? ';' : ':',
                                      unmet_dependencies[i] );
                 }
+                if ( missing_unmet_dependencies )
```

Comment:
Style: `if(` with no space in-between (I know our style is very non-standard, sorry, hopefully we'll switch to a more standard style at some point).

Reply:
Yes I have to be more careful as I am used to have a space after an if. Will fix that.